src/main/kotlin/org/ptss/support/core/mappers/RoleMapper.kt

```diff
@@ -0,0 +1,11 @@
+package org.ptss.support.core.mappers
+
+import io.quarkus.security.UnauthorizedException
+import org.ptss.support.domain.enums.Role
+
+object RoleMapper {
```

Comment:
Liever niet een hele nieuwe mapper voor één use case. Zorg ervoor dat er een fromString function in the Role enum class zit en gebruik die.